models/trigger_map.go

```diff
@@ -0,0 +1,85 @@
+package models
+
+import "fmt"
+
+// TriggerMapModel ...
+type TriggerMapModel []TriggerMapItemModel
+
+// Validate ...
```

Comment:
nit: we can clean up these dummy comments, we are not using golint anymore.

Reply:
Fixed here: https://github.com/bitrise-io/bitrise/pull/905/commits/08c5f009771caac0f336d9a28c1eef0eabf114ed